dtrace_probe.cc

```diff
@@ -59,8 +61,17 @@ namespace node {
       return Nan::Undefined();
     }
 
+    int cblen = argsinfo.Length() - fnidx - 1;
+    Local<Value> *cbargs = new Local<Value>[cblen];
```

Comment:
Ah sorry, I'm so used to implementing native bindings for functions with a fixed number of arguments that I forgot for a moment while thinking about that heap allocation that this change is about passing an arbitrary number of arguments, so that makes sense.